tests/tests-hz38/src/main/java/com/hazelcast/simulator/tests/quorum/QuorumCacheTest.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.simulator.tests.quorum;
+
+import com.hazelcast.quorum.QuorumException;
+import com.hazelcast.simulator.test.AbstractTest;
+import com.hazelcast.simulator.test.BaseThreadState;
+import com.hazelcast.simulator.test.annotations.Setup;
+import com.hazelcast.simulator.test.annotations.TimeStep;
+import org.junit.Assert;
+
+import javax.cache.Cache;
+import javax.cache.CacheManager;
+
+import static com.hazelcast.simulator.tests.icache.helpers.CacheUtils.createCacheManager;
+
+/**
+ * The QuorumCacheTest can be used to verify the Quorum behavior wherein
+ * a continuous tests will assert the
+ * 1. Adding member
+ * 2. Removing member
+ *    and the tests will pass or fail accordingly.
+ * */
+
+public class QuorumCacheTest extends AbstractTest {
```

Comment:
The javadoc can also explain that the operations are continuous and the grace period behaviour (how is it detected, what happens during the grace period, etc.)